action.yml

```diff
@@ -47,7 +47,9 @@ runs:
   steps:
     - id: helm-install
       name: Install a Helm chart
-      uses: docker://public.ecr.aws/l7o7z1g8/actions/helm-install-action:1.0.0
+      # TODO: revert to using the public registry
+      #uses: docker://public.ecr.aws/l7o7z1g8/actions/helm-install-action:${{ action.scm.sha }}
+      uses: docker://registry.saas-dev.beescloud.com/staging/helm-install-action:${{ action.scm.sha }}
```

Comment:
This needs to be solved before this change can be released. For now I am leaving it in here to show that the PR build works.

Reply:
After we saw the build succeed and the image from this PR was promoted to the public repo using the tag `1.1.0`, I changed the action.yml back to using the public registry and the new tag.